components/scream/data/scream_input.yaml

```diff
@@ -18,4 +18,118 @@ Atmosphere Driver:
     Reference Grid: Physics GLL
     Dynamics Driven:
       Dynamics Namelist File Name: ./data/theta.nl
+
+SCREAM:
+
+  #=====================
+  # I'm confused how we should set this. You shouldn't be able
+  # to change this once a run starts, but you should be able to choose it
+  # when you start a new run... What if your initial condition is for year 2000
+  # but you're doing a control run and want it to start at 01-01-0001?
+  Start Date: 01-01-2000
+
+  # Model Timesteps
+  whole_model_dt: 3600.0 #aka phys/dyn coupling dt
+
+  # Number of sub or supersteps of whole_model_dt for given proc.
+  # maybe have positive #s be supersteps and negative numbers being
+  # number of substeps? This might require a generalization of substepping
+  # since everything except rad always substeps and rad always supersteps?
+  # There's also an issue here where AMB just
+  # rejiggered dyn's substepping to be ~resolution invariant. Maybe
+  # use that impl for all substeps?
+  n_turb_steps: 1
+  n_micro_steps: 1
+  n_rad_steps: 1
+  dt_remap_factor: 2
+  dt_tracer_factor:  6
+
+  Tuning Parameters:
+    SHOC:
+      Ckh_s_def: 0.1 #using "moderate-stable"
+      Ckm_s_def: 0.1 #vals here as used by
+      Ck_s_min: 0.1  #DYAMOND2 SCREAMv0 run
+      #there will definitely eventually be more
+
+    P3:
+      mu_r_constant: 1.0
+      p3_qc_accret_expon: 1.15
+      p3_qc_autocon_expon: 2.47
+      #there will definitely eventually be more
```

Comment:
We should create a git-issue after this PR goes in to have the respective teams audit the list of tuning parameters and add more if needed.